packages/grafana-data/src/panel/registryFactories.ts

```diff
@@ -58,7 +67,19 @@ export function createFieldConfigRegistry<TFieldConfigOptions>(
     }
 
     registry.register(fieldConfigProp);
+
+    if (fieldConfigProp.category && standardOptionsExtensions[fieldConfigProp.category[0]]) {
+      for (let extensionProperty of standardOptionsExtensions[fieldConfigProp.category[0]]) {
+        registry.register(extensionProperty);
+      }
+    }
   }
 
   return registry;
 }
+
+function isStandardConfigExtension(property: FieldConfigPropertyItem, standardProperties: FieldConfigPropertyItem[]) {
+  return Boolean(
+    standardProperties.find((p) => property.category && p.category && property.category[0] === p.category[0])
```

Comment:
Thinking about deprecating category as an array - seems like we are not using it anywhere this way, always a single string. @torkelo do you recall any nested categories somewhere?

Reply:
don't think so, also been thinking of making this string, and an id. And elsewhere be able to add category info objects with id, name, description & sort order . With a category info object we could more easily control sorting, add description, and maybe help links & tips